Force all python scripts to explicitly use Python 3.7 instead of the default one.

This is our internal change no intended for upstream.

--- libevent-2.1.11-stable/event_rpcgen.py
+++ libevent-2.1.11-stable/event_rpcgen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3.7
 #
 # Copyright (c) 2005-2007 Niels Provos <provos@citi.umich.edu>
 # Copyright (c) 2007-2012 Niels Provos and Nick Mathewson

--- libevent-2.1.11-stable/test/check-dumpevents.py
+++ libevent-2.1.11-stable/test/check-dumpevents.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3.7
 #
 # Post-process the output of test-dumpevents and check it for correctness.
 #
